apps/backoffice-v2/src/pages/Entity/components/Case/hooks/usePendingRevisionEvents/utils/calculate-workflow-revisionable-event.tsx

```diff
@@ -0,0 +1,20 @@
+import { TWorkflowById } from '@/domains/workflows/fetchers';
+import { getPostDecisionEventName } from '@/lib/blocks/components/CallToActionLegacy/hooks/useCallToActionLegacyLogic/useCallToActionLegacyLogic';
+import { CommonWorkflowEvent } from '@ballerine/common';
+
+export const calculateWorkflowRevisionableEvent = (
+  workflow: TWorkflowById,
+  documentStatus: string,
```

Comment:
In general for types around context and documents `DefaultContextSchema` is now pretty accurate. Here you'd see `documentStatus` is "revision" | "approve" and so on. No change required, something to have in mind for the future.